### PR TITLE
Bug 1101793 - delete single quote in first layer of german keyboard

### PR DESCRIPTION
**Pull request by Gioyik**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[Gioyik Giovanny Andres Gongora Granada (Gioyik)](https://github.com/Gioyik) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ae814c788fd8)

File: apps/keyboard/js/layouts/de.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ Keyboards.de = {
     ], [
       { value: 'a' }, { value: 's' }, { value: 'd' }, { value: 'f' },
       { value: 'g' } , { value: 'h' }, { value: 'j' }, { value: 'k' },
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/de.js]: extra space before `,`, this is not related to your changes, but you could help fix them if you want to.

Thanks.

**Comment by try-server-hook** [comment]
[Gioyik Giovanny Andres Gongora Granada (Gioyik)](https://github.com/Gioyik) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=928f49b9dd62)

**Comment by try-server-hook** [comment]
[Gioyik Giovanny Andres Gongora Granada (Gioyik)](https://github.com/Gioyik) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=43e074f39b60)

**Comment by try-server-hook** [comment]
[Gioyik Giovanny Andres Gongora Granada (Gioyik)](https://github.com/Gioyik) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=aec9732bc4ed)

**Pull request merged**